docker-compose.yaml

```diff
@@ -0,0 +1,59 @@
+version: '3.7'
+
+services:
+
+  usaspending-db:
+    container_name: usaspending-db
+    image: postgres:10.6-alpine
+    volumes:
+      - type: bind
+        source: ${POSTGRES_CLUSTER_DIR}
+        target: /var/lib/postgresql/data
+    restart: on-failure:3 # Will attempt to start the db 3 times
+    ports:
+      - 5432:5432
+    environment:
+      POSTGRES_USER: ${POSTGRES_USER}
+      POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
+      POSTGRES_DB: data_store_api
+
+  usaspending-db-migrate:
+    build: .
+    volumes:
+     - .:/dockermount
+    command: /bin/sh -c "python -u manage.py migrate"
+    environment:
+      DATABASE_URL: postgres://${POSTGRES_USER}:${POSTGRES_PASSWORD}@usaspending-db:5432/data_store_api
+
+  usaspending-db-sql:
+    build: .
+    volumes:
+     - .:/dockermount
+    command: >
+      /bin/sh -c "mkdir $$MATVIEW_SQL_DIR &&
+      python -u usaspending_api/database_scripts/matview_generator/matview_sql_generator.py --dest=$$MATVIEW_SQL_DIR"
+    environment:
+      MATVIEW_SQL_DIR: ${MATVIEW_SQL_DIR}
+
+  usaspending-db-init:
+    image: postgres:10.6-alpine
+    volumes:
+     - .:/dockermount
+    command: >
+      /bin/sh -c "cd /dockermount &&
+      psql $$DATABASE_URL -c 'CREATE ROLE readonly' || true &&
+      psql $$DATABASE_URL -f usaspending_api/database_scripts/matviews/functions_and_enums.sql &&
+      cat $$MATVIEW_SQL_DIR/*.sql | psql $$DATABASE_URL -f -"
```

Comment:
`cat $$MATVIEW_SQL_DIR/*.sql | psql $$DATABASE_URL -v ON_ERROR_STOP=1 -f -"`

Reply:
It would be better to move the matviews to a separate command, and add the `load_reference_data` management command to this docker command.

Reply:
How are we going to control which task gets executed when? I don't see any of the sleeps in the compose file anymore